scripts/install/README.md

```diff
@@ -0,0 +1,101 @@
+# Installation Script for Spartacus
+
+Installation script does the following:
+
+- Build Spartacus libraries from source
+- Create a new shell app for the Spartacus libraries
+- Install the libraries in the new shell app
+- Add custom configurations to the shell app
+
+It can also:
+
+- Install spartacus in production mode
+- Install Spartacus in SSR mode
+
+## How to install
+
+Installation directory for SpartacusScript's output directory is set as below (related to script's root, `./scripts/install`):
+
+```bash
+# top directory for the installation output (must be outside of the project)
+if [ -z $BASE_DIR ]; then
+    BASE_DIR="../../../spartacus-${SPARTACUS_VERSION}"
+fi
+```
+
+so it's one level above the spartacus project root. To change the output directory `$BASE_DIR` can be overriden in the `config.sh` file.
+
+All script's commands should be run while being in it's root directory:
+
+```bash
+cd ./scripts/install
+```
+
+Before running the main script, please ensure that:
+
+- line `//localhost:4873/:_authToken="VNOlucnBmPrXVjzHVEHtWQ=="` is added to the `~/.npmrc` file
+- `ANGULAR_CLI_VERSION` variable is set properly in the `config.sh` file (it must meet project's `package.json` `@angular/cli` version requirement)
+- `OCC_PREFIX` variable is set to proper value (depends on backend version)
+- adjust any other variables available in the `config.default.sh` if needed (by setting them up in the `config.sh`)
+
+Once ready start the installation as below:
+
+```bash
+ ./run.sh install
+```
+
+The script will build the libraries from source, create a new shell app and install the libraries in it.
+
+By default, Spartacus will be built from the `develop` branch. You can specify a different branch in the `config.sh` file.
+
+The installation will create two folders in the `$BASE_DIR` directory:
+
+- `apps` (generated/executable apps)
+- `clone` (cloned Spartacus repo)
+
+## Starting Spartacus
+
+```bash
+./run.sh start
+```
+
+This command starts a server for Spartacus, which is under `"${BASE_DIR}/apps/csr"` by default.
+
+## Stop Spartacus
+
+```bash
+./run.sh stop
+```
+
+This command stops the running Spartacus PM2 processes.
+
+## Update Spartacus
+
+Just run the installation again (stop it first if it's running).
+
+```bash
+./run.sh stop
+./run.sh install (script ask you to rename/delete the `$BASE_DIR` if it exists)
```

Comment:
It will ask or it just removes the directory?

Reply:
It will remove, it's the old message.